cqlsh_tests/test_cqlsh.py

```diff
@@ -1040,7 +1040,7 @@ def test_default_keyspaces_exist(self):
         node1, = self.cluster.nodelist()
 
         # Describe keyspaces
-        expected_keyspaces = ['system_schema', 'system', 'system_traces', 'system_views',
+        expected_keyspaces = ['system_schema', 'system', 'system_traces', 'system_views', 'system_metrics',
```

Comment:
@Mmuzaf this will not work because system_metrics will be only on trunk and this test will be executed for 4.0+ so whey 4.0 will be built, this one will not be there. You need to do it like below for system_cluster_metadata case:

        if self.cluster.version() >= LooseVersion('5.1'):
            expected_keyspaces.append('system_metrics')

Reply:
Makes sense, fixed.